.travis.yml

```diff
@@ -1,4 +1,16 @@
 sudo: false
 
-language: node_js
-node_js: stable
+language: java
```

Comment:
Changing language is not necessary, could you just keep the `jdk` property in addition ?